Source/Scene/ModelExperimental/ModelExperimentalSceneGraph.js

```diff
@@ -450,6 +494,43 @@ ModelExperimentalSceneGraph.prototype.updateModelMatrix = function () {
   }
 };
 
+const scratchObjectSpace = new Matrix4();
+
+ModelExperimentalSceneGraph.prototype.updateJointMatrices = function () {
+  const model = this._model;
+  const skinnedNodes = model._skinnedNodes;
+
+  for (let i = 0; i < skinnedNodes.length; i++) {
+    const node = skinnedNodes[i];
+
+    const objectSpace = Matrix4.inverseTransformation(
+      node.transformToRoot,
+      scratchObjectSpace
+    );
+
+    const computedJointMatrices = node.computedJointMatrices;
+    const joints = node.joints;
+    const inverseBindMatrices = node.inverseBindMatrices;
+
+    for (let m = 0; m < inverseBindMatrices.length; ++m) {
+      // [joint-matrix] = [node-to-root^-1][joint-to-root][inverse-bind]
```

Comment:
minor formatting tweak

```suggestion
      // jointMatrix = nodeToRoot^(-1) * jointToRoot * inverseBind
```